src/watchdog/observers/winapi.py

```diff
@@ -119,11 +119,11 @@ def _errcheck_dword(value, func, args):
     return args
 
 
-kernel32 = ctypes.WinDLL("kernel32")
+kernel32 = ctypes.WinDLL("kernel32")  # type: ignore[attr-defined]
```

Comment:
With this being Windows only we could again encapsulate the whole file with a platform check.  I mean yeah, it's ugly.  I take a layer of indentation over lack of good hint checking though, personally.